core/src/main/scala/org/ensime/indexer/ClassfileIndexer.scala

```diff
@@ -63,10 +63,18 @@ trait ClassfileIndexer {
         interfaces.toList.map(ClassName.fromInternal),
         Access(access),
         (ACC_DEPRECATED & access) > 0,
-        Nil, Nil, RawSource(None, None)
+        Nil, Nil, RawSource(None, None),
+        isScala = false
       )
     }
 
+    override def visitAttribute(attr: Attribute): Unit = {
+      val attrType = attr.`type`
+      if (attrType == "Scala" || attrType == "ScalaSig") {
+        clazz = clazz.copy(isScala = true)
```

Comment:
cool, is this more reliable than looking in the source file name? If debugging symbols or stripped it's hard to tell.